bot/exts/info/source.py

```diff
@@ -83,12 +96,12 @@ async def build_embed(self, source_object: SourceType) -> Optional[Embed]:
         elif isinstance(source_object, commands.Command):
             description = source_object.short_doc
             title = f"Command: {source_object.qualified_name}"
-        elif isinstance(source_object, TagIdentifier):
-            title = f"Tag: {source_object}"
-            description = ""
-        else:
+        elif issubclass(type(source_object), commands.Cog):
```

Comment:
Closes #2022

Previously the error prompted because `isinstance` was returning False when an `TagIdentifier` object was compared to itself. Instead of checking that, we check if the source object `issubclass` of commands.Cog and pass the TagIdentifier to else.